### ISSUE TITLE
Bug when updating resource.

### ISSUE THREAD

**Issue by gskip17** [periscope-ps/UNISrt#12]
```
Trouble Updating Unis Host Topology... Continuing
Traceback (most recent call last):
  File "/user/gskipper/topology/ryu/osiris_main.py", line 457, in check_update_host_topology
    topology.domains[index] = most_recent_domain
  File "/usr/local/lib/python3.5/site-packages/unisrt-1.0.dev0-py3.5.egg/unis/models/models.py", line 41, in __setitem__
    return self._obj._setitem(i, v, self._rt)
  File "/usr/local/lib/python3.5/site-packages/lace-0.2.0-py3.5.egg/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/unisrt-1.0.dev0-py3.5.egg/unis/models/models.py", line 173, in _setitem
    self._update(self._rt_reference)
  File "/usr/local/lib/python3.5/site-packages/lace-0.2.0-py3.5.egg/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
TypeError: _update() missing 1 required positional argument: 'ctx'
```

**Comment by jemusser** [comment]
Resolved in 17ec5397817db20296342a48901773ccf270f7f0

**Comment by jemusser** [comment]
Issue re-appeared in another function, resolved in 5e46818618d8a4f0e091c8024fa904ed75acc088
